luatest/assertions.lua

```diff
@@ -466,6 +471,7 @@ local function _assert_error_msg_equals(stripFileAndLine, expectedMsg, func, ...
             prettystr(error_msg), prettystr(expectedMsg))
         failure(failure_message, nil, 3)
     end
+    log.info('Assert error message %s equals %s', error_msg, expectedMsg)
```

Comment:
I propose to use the following format:

`` Assert error message `%s` equals to `%s` ``

Also, looks like it should be at the very beginning of the function.

Reply:
I've fixed that